tests/emulator/src/dpi.cc

```diff
@@ -33,6 +40,7 @@ void VBridgeImpl::dpiDumpWave() {
 }
 
 [[maybe_unused]] void dpiInitCosim() {
+  std::signal(SIGINT, sigint_handler);
```

Comment:
According to https://stackoverflow.com/a/1641223, `sigaction` seems to be a more reliable way.

Reply:
`std::signal` is an ISO C++ standard, while `sigaction` is POSIX standard. Hard to say which is more reliable.